views/index.ejs

```diff
@@ -38,11 +38,15 @@
     <div id="react_root"></div>
     <div class="react_modal__container"></div>
     <div class="top_bar">
-      <form onsubmit="return false" novalidate class="search_form"
-        <% if(config.app.versionFlag) { %>
-          data-flag-text="<%= config.app.versionFlag %>"
-        <% } %> >
-        <div class="search_form__logo"></div>
+      <form onsubmit="return false" novalidate class="search_form">
+        <button
```

Comment:
Let's add a title to it like we did on other buttons. Especially as there is no inner content.
What about `title="<%= _('Qwant Maps home') %>"`? Or `'Back to home'` ?

Reply:
wdyt about just "Qwant Maps" ?

Reply:
The goal is to describe the action. "Qwant Maps" seems quite ambiguous. But we can ask the design team.

Reply:
I asked the design team. For now, we decided to go with your proposition `Qwant Maps Home`